.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+- package-ecosystem: github-actions
```

Comment:
Is there a plan to have dependabot manage python deps, too?

Reply:
Not that I am aware of. But I remember that Patrick mentioned another tools for updating the dependencies somewhere else (renovate?). I would suggest to do it step by step (so not in this PR).

Reply:
Dependabot currently supports poetry-mediated python dependency management [via the `pip` configuration](https://docs.github.com/en/code-security/dependabot/ecosystems-supported-by-dependabot/supported-ecosystems-and-repositories#pip-and-pip-compile).

[uv support](https://github.blog/changelog/2025-03-13-dependabot-version-updates-now-support-uv-in-general-availability/) has recently been announced.

I suggest adding in a config section for watching python deps since you're adding in GHA support in this PR.